src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+const VeteranInformationComponent = props => {
+  let dateOfBirthFormatted = '';
+  let genderFull = '';
+  if (props.user && props.user.dob) {
+    dateOfBirthFormatted = moment(props.user.dob).format('MMMM Do YYYY');
+  }
+  if (props.user && props.user.gender === 'M') {
+    genderFull = 'Male';
+  } else if (props.user && props.user.gender === 'F') {
+    genderFull = 'Female';
+  }
+  const alertContent = (
+    <>
+      <p>
+        <strong>
+          {props.user.userFullName.first} {props.user.userFullName.last}
+        </strong>
+      </p>
+      <p>Date of birth: {dateOfBirthFormatted}</p>
+      <p>Gender: {genderFull}</p>
+    </>
+  );
+  return (
+    <div>
+      <p>This is the information we have on file for you.</p>
+      <AlertBox content={alertContent} status="info" isVisible />
+      <p>
+        <strong>Note:</strong> If you need to update your personal information,
+        please call Veterans Benefits Assistance at{' '}
+        <a href="tel:800 827 1000">800-827-1000</a> between 8:00 a.m. and 9:00
```

Comment:
There's a cool [pull request](https://github.com/department-of-veterans-affairs/vets-website/pull/12546) in the works to standardize phone numbers eventually. 

For the time being though, a way to make phone numbers a little more accessible is to add an aria-label with separated digits, so something like `aria-label= 800. 8 2 7. 1000.`.

Reply:
I'm going to close that PR & open a new one in the VFST repo instead of this one. So I don't know when it'll get approved and available.

Reply:
oh, Jennifer told me not to add an aria label for a phone number because then the number gets read twice

Reply:
fixed this